application/controllers/admin/DataEntry.php

```diff
@@ -894,7 +894,7 @@ public function editdata($subaction, $id, $surveyid)
                         $thisqid = $fname['qid'];
                         $currentvalues = array();
                         $rawvalues = [];
-                        $myfname = $fname['sid'] . 'X' . $fname['gid'] . 'X' . $fname['qid'];
+                        $myfname = 'Q' . $fname['qid'];
```

Comment:
The new columns name are Qqid ? or it's the input name ?

Reply:
@Shnoulle no issue with the question, I just provided a function that will provide meaningful output even for cases that I did not touch in my response, as I did not write a full documentation yet.